front-end/src/components/league-page/leaderboard/leaderboardCard.tsx

```diff
@@ -0,0 +1,41 @@
+import { SxProps, Theme,  useTheme } from '@mui/material';
+import { Box } from '@mui/system';
+import { getGradient } from '../general';
+import { darken } from '@mui/system/colorManipulator';
+import React from 'react';
+import { LeaderboardRowProps, Row } from './row';
+
+
+interface LeaderboardCardProps {
+    sortedRowProps: LeaderboardRowProps[]
+    header: LeaderboardRowProps,
+    title: React.ReactNode,
+    sx?: SxProps<Theme> | undefined;
+    width: number,
+    height: number
+}
+
+export default function LeaderboardCard(props: LeaderboardCardProps) {
+  const theme = useTheme() as Theme;
+
+  return (
+    <Box sx={{
+      minWidth: props.width,
+      minHeight: props.height,
+      flexGrow: 1,
+      flexWrap: 'wrap',
+    }}>
+      <Box sx={{ p: 0.2, borderRadius: '4px 4px 0px 0px', background: getGradient(theme.palette.risenBoxBg.main)  }}>
+        {props.title}
+        <hr/>
+      </Box>
+      <Box sx={{ background: getGradient(theme.palette.risenBoxBg.main) }}>
```

Comment:
Just wondering, why not just use a table for this?

Reply:
Table didnt have the look that i was going for so i just built the component myself.

Reply:
Ok, ideally this can be moved out into its own full component so it can be reused then